db_stress_tool/db_stress_test_base.cc

```diff
@@ -2358,6 +2375,19 @@ Status StressTest::TestGetPropertiesOfAllTables() const {
   return db_->GetPropertiesOfAllTables(&props);
 }
 
+void StressTest::TestGetUpdatesSince(ThreadState* thread) {
+  SequenceNumber latest_seqno = db_->GetLatestSequenceNumber();
+  SequenceNumber seq = thread->rand.Next64() % latest_seqno + 1;
+  std::unique_ptr<TransactionLogIterator> iter;
+  Status s = db_->GetUpdatesSince(seq, &iter);
```

Comment:
Doesn't the user have to set some option for this to return results that make sense?

Reply:
Yeah - the wal_xxx parameters ned to be set very high. Still it was a bit flaky in my local run. Let me try again and add `TestGetUpdatesSince` in another PR for easier testing.